deploy/architecture.md

```diff
@@ -4,13 +4,52 @@
 
 See [introduction](../build/pooling.md#introduction)
 
-## Architecture
+## Definitions and terminology notes
 
-See [architecture](../build/deploy/README.md#architecture)
+This section defines some concepts and related references used in this documentation.
+
+### DSS Region
+
+A DSS Region is a region in which a single, unified airspace representation is
+presented by one or more interoperable DSS instances, each instance typically
+operated by a separate organization.  A specific environment (for example,
+"production" or "staging") in a particular DSS Region is called a "pool".
+
+### DSS Pool
+
+A DSS Pool is a set of interoperable and interconnected DSS instances in a specific
+DSS Region. Each instance is typically operated by a separate organization.
+
+### DSS instance
+
+A DSS instance is a single logical replica in a DSS pool hosted by a single
+organization.
 
-### Terminology notes
+### Pooling
 
-See [teminology notes](../build/pooling.md#terminology-notes).
+The process required by a DSS Instance to join a DSS Pool is referred to "Pooling"
+in this documentation.
+
+### CRDB cluster
```

Comment:
- Add 'CRDB node'?
- What's the relationship between CRDB cluster/node and DSS instance/pool?